pkg/apis/submariner.io/v1/types.go

```diff
@@ -90,8 +91,8 @@ type GatewayList struct {
 type HAStatus string
 
 const (
-	HAStatusActive  HAStatus = "ACTIVE"
-	HAStatusPassive HAStatus = "PASSIVE"
+	HAStatusActive  HAStatus = "active"
```

Comment:
Those where changed while playing with labels, full upcase was weird compared to other resources.